src/roulette.py

```diff
@@ -126,3 +126,380 @@ def get(self, bin: int) -> Bin:
+
+        :param wheel: The Wheel object to which the five bet outcomes will be added.
+        :type wheel: :class:`Wheel`
+
+
+        **Note**: The wheel is assumed to have bins for the numbers 0, 00, 1, 2, 3, and an extra
+        bin at index 37 for the "00" outcome.
+        """
+        odds_for_five_bet = 6
+        five_bet_name = "00-0-1-2-3"
+        five_bet_outcome = Outcome(five_bet_name, odds_for_five_bet)
+        bin_indexes_for_five_bet = {0, 1, 2, 3, 37}
+
+        for index in bin_indexes_for_five_bet:
+            wheel.addOutcome(index, five_bet_outcome)
+
+    @staticmethod
+    def build_bins_for_even_money_bets(wheel: Wheel) -> None:
+        """
+        Populates **Bin** the given **wheel** with even money bet outcomes, each associated with
+        even money bets in roulette.
+
+        The function adds even money bet outcomes for Red, Black, Low, High, Even, and Odd bets,
+        each with an associated odds value of 1.
+        - Red and Black outcomes correspond to bets on the color of the number.
+        - Low and High outcomes correspond to bets on numbers 1-18 and 19-36, respectively.
+        - Even and Odd outcomes correspond to bets on even and odd numbers, respectively.
+
+        :param wheel: The Wheel object to which the even money bet outcomes will be added.
+        :type wheel: :class:`Wheel`
+
+        **Note**: The wheel is assumed to have bins for all possible numbers.
+        """
+        even_bet_odds = 1
+
+        red_bet_name = "Red"
+        black_bet_name = "Black"
+        low_bet_name = "Low"
+        high_bet_name = "High"
+        even_bet_name = "Even"
+        odd_bet_name = "Odd"
+
+        even_bet_numbers = set(range(1, 37))
+        red_bet_numbers = {
+            1,
+            3,
+            5,
+            7,
+            9,
+            12,
+            14,
+            16,
+            18,
+            19,
+            21,
+            23,
+            25,
+            27,
+            30,
+            32,
+            34,
+            36,
+        }
+
+        red_bet = Outcome(red_bet_name, even_bet_odds)
+        black_bet = Outcome(black_bet_name, even_bet_odds)
+        low_bet = Outcome(low_bet_name, even_bet_odds)
+        high_bet = Outcome(high_bet_name, even_bet_odds)
+        even_bet = Outcome(even_bet_name, even_bet_odds)
+        odd_bet = Outcome(odd_bet_name, even_bet_odds)
+
+        for number in even_bet_numbers:
+            if number < 19:
+                wheel.addOutcome(number, low_bet)
+            if number >= 19:
+                wheel.addOutcome(number, high_bet)
+            if number % 2 == 0:
+                wheel.addOutcome(number, even_bet)
+            if number % 2 != 0:
+                wheel.addOutcome(number, odd_bet)
+            if number in red_bet_numbers:
+                wheel.addOutcome(number, red_bet)
+            if number not in red_bet_numbers:
```

Comment:
can be replaced with an 'else'. that would be more optimized as can avoid the lookup

Reply:
Done.